manifests/c/Chocolatey/Chocolatey/2.2.2.0/Chocolatey.Chocolatey.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Chocolatey.Chocolatey
+PackageVersion: 2.2.2.0
+InstallerLocale: en-US
+Scope: machine
+AppsAndFeaturesEntries:
+- DisplayName: Chocolatey (Install Only)
+  UpgradeCode: '{62FC4E65-47F0-488A-B841-ECA07E98A58B}'
+Installers:
+- Architecture: x86
+  InstallerType: wix
+  InstallerUrl: https://github.com/chocolatey/choco/releases/download/2.2.2/chocolatey-2.2.2.0.msi
+  InstallerSha256: 26D49421F65D074C6F5D6C6EBD3353D9FAF5764D7E0C27752BB7B4429D41DEF2
+  ProductCode: '{19B4B604-DC1C-4CEA-B5A4-688820F1AB72}'
+ManifestType: installer

```

```suggestion
  ProductCode: '{19B4B604-DC1C-4CEA-B5A4-688820F1AB72}'
  ExpectedReturnCodes:
  - InstallerReturnCode:  1603
    ReturnResponse: alreadyInstalled
    ReturnResponseUrl: https://ch0.co/cli-msi
ManifestType: installer
```

The MSI was updated to prevent upgrading with the MSI as it is not supported. The expected exit code should be added here to provide some explanation to the user.